README.md

```diff
@@ -4,6 +4,16 @@ A repository for developing the User Interface for receiving data and controllin
 
 ## Packages Info
 
+### Whole Launch
+
+1.	Open your terminal and navigate to your working directory (for example, ros2_ws).
+2.	Clone the latest version of the MRCS: `git@github.com:GonzagaRobotics/ManualRoverControlSystem.git`
+3.	Once the repository is cloned, navigate to the top of the ros2_ws directory and build the project using Colcon by running the command: `colcon build`. If an error occurs, try again until it works.
```

Comment:
This is 2 actions, so break this up into 2 tasks. Use verbs as the first word and keep each instruction short.
For example:
1. Navigate to `/path/to/ManualRoverControlSystem/ros2_ws/`
1. Build the project by running `colcon build`
    - If you get a <insert specific, reproducible error here> error, refer to <Reference to section on common issues>
  
 Note: Yes you can link to sections in your readme in markdown, which you can create with \# symbols. Here's how: https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown. Markdown links work by putting \[Text to display\]\(link\), where link is a web address or a reference to a header in the markdown document (or even in other markdown files)